sched/Kconfig

```diff
@@ -973,12 +973,6 @@ config SCHED_INSTRUMENTATION_CPUSET
 	---help---
 		Monitor only CPUs in the bitset.  Bit 0=CPU0, Bit1=CPU1, etc.
 
-config SCHED_INSTRUMENTATION_HIRES
```

Comment:
remove SCHED_INSTRUMENTATION_HIRES in the whole code base.

Reply:
All removed, others in this PR： https://github.com/apache/nuttx-apps/pull/1593

Reply:
but here has one left: https://github.com/apache/nuttx/blob/master/Documentation/guides/tasktraceuser.rst